docs/Usage/Identity.md

```diff
@@ -0,0 +1,196 @@
+# Identity API Usage
+
+This document details the APIs provided by Identity extension, along with sample code snippets on how to properly use the APIs.
+
+For more in-depth information about the Identity extension, visit the [official SDK documentation on Identity](https://developer.adobe.com/client-sdks/documentation/mobile-core/identity/).
+
+#### Getting Identity extension version
+
+##### Java
+
+```java
+final String identityExtensionVersion = Identity.extensionVersion();
+```
+
+##### Kotlin
+
+```kotlin
+val identityExtensionVersion: String = Identity.extensionVersion()
+```
+
+
+#### Registering Identity extension
+`Identity.EXTENSION` represents a reference to the IdentityExtension class that can be used to register
```

Comment:
Can you reword this sentence?

Reply:
`Identity.EXTENSION` represents a reference to the `IdentityExtension` class that can be registered with `MobileCore` via its `registerExtensions` api.